src/page/address-step.page.ts

```diff
@@ -0,0 +1,13 @@
+import { $, ElementFinder } from 'protractor';
+
+export class AddressStepPage {
+  private proceedToCheckout: ElementFinder;
```

Comment:
the names of the functions are fine as they represent the action to be made on the page, yet the naming for these properties can be improved so it's easier to read where is the action being performed if you take a look at the description of the readme the example names the property as `Menu` so you know you're clicking on a menu
![image](https://user-images.githubusercontent.com/1211491/124609725-815e3580-de35-11eb-9f2d-65edae3b46ed.png)

let's improve the naming for this and the other properties of the other pags

Reply:
Done